packages/editor/src/components/entities-saved-states/index.js

```diff
@@ -197,10 +196,12 @@ export default function EntitiesSavedStates( { close } ) {
 					{ __( 'Save' ) }
 				</Button>
 				<Button
-					icon={ closeIcon }
+					className="editor-entities-saved-states__cancel-button"
 					onClick={ dismissPanel }
-					label={ __( 'Close panel' ) }
```

Comment:
I wonder if we should keep this explicit label indicating that it's closing the panel. Could use some inputs from accessibility folks. WFYT @tellthemachines ?